### DISCUSSION TITLE
Extend gateway settings

### DISCUSSION THREAD

**Discussion by timrvv** [impress-org/givewp#7065] [Developer Q&A]
How can we extend gateway settings (see screenshot below how it's done for Stripe) to support parameters such as merchantId, terminalId, any other values we would like to pass to the gateway?

![stripe_settings](https://github.com/impress-org/givewp/assets/10048460/cc9b1d8a-5b5e-4a48-9ae3-79fbdf87ed9a)

**Comment by jonwaldstein** [comment]
Hey @timrvv,

Is your goal to send these additional parameters to Stripe during a donation?
